app/decorators/physical_switch_decorator.rb

```diff
@@ -0,0 +1,42 @@
+class PhysicalSwitchDecorator < MiqDecorator
+  def self.fonticon
+    'ff ff-network-switch'
+  end
+
+  def quadicon
+    {
+      :top_left     => {
+        :text    => (hardware.physical_ports ? hardware.physical_ports.size : 0),
+        :tooltip => 'Ports'
+      },
+      :top_right    => {
+        :tooltip => power_state.try(:downcase),
```

Comment:
Just being curious, is there really a powerstate for a physical switch? :confused:

Reply:
Yes, the LXCA returns information about the switch power state. @rodneyhbrown7, do you have more information about?

Reply:
I talked to @rodneyhbrown7 and he said that it is ok to have a powerstate for switches.